### PR TITLE
Update README with more friendly sample steps

### PR DESCRIPTION
**Pull request by magicmonty**
The sample in the README was a bit incomplete. I needed to fiddle around to get it running.
It was missing the needed references for the main project and also the resulting window looked quite ugly on first run, so I tweaked the XAML a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,29 +75,30 @@ See the [SingleCounter](https://github.com/elmish/Elmish.WPF/tree/master/src/Sam
 
    The strings identify the binding names to be used in the XAML views. The [Binding module](https://github.com/elmish/Elmish.WPF/blob/master/src/Elmish.WPF/Binding.fs) has many functions to create various types of bindings.
 
-6. Create a WPF user control library project to hold you XAML files, add a reference to this project from your Elmish project, and define your views and bindings in XAML:
+8. Create a WPF user control library project to hold you XAML files, add a reference to this project from your Elmish project, and define your views and bindings in XAML:
 
    ```xaml
    <Window
```
Comment (cmeeren) [inline README.md]: The XAML sample in the readme is "optimized" for clarity of example while reading the readme, not renderability. (The reader has already been referred to the SingleCounter sample and told that these are just the central points.) I'd like to keep it the way it was. If you want, feel free to add `(kept simple here for clarity)` or something to that effect before the ending colon on this point.

File: README.md [hunk 2]
```diff
@@ -70,29 +75,30 @@ See the [SingleCounter](https://github.com/elmish/Elmish.WPF/tree/master/src/Sam
...
    ```
 
-7. Add the entry point to your console project:
+9. Add the entry point to your console project:
 
    ```F#
    open System
    open Elmish
-   
+
```
Comment (cmeeren) [inline README.md]: Undo whitespace edit

File: README.md (hunk 1 above)
Comment (magicmonty) [reply README.md]: Good point, will change it back

**Comment by magicmonty** [comment]
@cmeeren Resolved all issues. Seems my Vim has removed the spaces on the empty lines ;)

**Comment by cmeeren** [comment]
Great! And thanks for squashing. :)

**Comment by magicmonty** [comment]
You're welcome :D

**Pull request merged**